workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -143,51 +143,29 @@ async def get_area_of_bpolys(bpolys: Dict):
                     )
             ) / (1000*1000) as area_sqkm
         """
-    polygon = json.dumps(bpolys["features"][0]["geometry"])
     async with get_connection() as conn:
-        result = await conn.fetchrow(query, polygon)
+        result = await conn.fetchrow(query, geojson.dumps(bpolys))
     return result["area_sqkm"]
 
 
-async def get_bpolys_from_db(
-    dataset: str, feature_id: Union[int, str], fid_field: str
-) -> geojson.FeatureCollection:
-    """Get bounding polygon from the geodatabase as a GeoJSON FeatureCollection."""
-    logging.info("(dataset, fid_field, id): " + str((dataset, fid_field, feature_id)))
+async def get_region_from_db(feature_id: Union[int, str], fid_field: str) -> Feature:
+    """Get regions from geodatabase as a GeoJSON Feature object"""
+    logging.info("(fid_field, id): " + str((fid_field, feature_id)))
 
     # Safe against SQL injection because of predefined values
     # (See oqt.py and definitions.py)
     query = (
-        """
-        SELECT json_build_object(
-            'type', 'FeatureCollection',
-            'crs',  json_build_object(
-                'type',      'name',
-                'properties', json_build_object(
-                    'name', 'EPSG:4326'
-                )
-            ),
-            'features', json_agg(
-                json_build_object(
-                    'type',       'Feature',
-                    'id',         {fid_field},
-                    'geometry',   public.ST_AsGeoJSON(geom)::json,
-                    'properties', json_build_object(
-                    )
-                )
-            )
-        )
-        FROM {dataset}
-        WHERE {fid_field} = $1
-    """
-    ).format(fid_field=fid_field, dataset=dataset)
+        "SELECT ST_AsGeoJSON(geom) "
+        + "FROM regions "
+        + "WHERE {0} = $1".format(fid_field)
```

Comment:
I think the `fid_field` can be directly passed from an API request to this point where the query is executed. We should have a sanity checking in the API implementation.

Reply:
fid_field is sanity checked in `oqt.py` using the function definied in `geodatabase/client.py`
```python
         if not db_client.sanity_check_fid_field(dataset, fid_field):
            raise ValueError("Input feature id field is not valid: " + fid_field)
```

Reply:
missed that, thanks :)